packages/special-pages/types/release-notes.ts

```diff
@@ -7,11 +7,146 @@
  * **DO NOT** edit this file directly as your changes will be lost.
  */
 
+/**
+ * Message sent from browser when release notes are updated
+ */
+export type UpdateMessage =
```

Comment:
As mentioned before - give the objects a title in the schema, and this type will be greatly improved.